R/utils.R

```diff
@@ -6,7 +6,21 @@ setup_log <- function(threshold = "INFO", file = "info.log") {
 
   return(invisible(NULL))
 }
-
+#' Set up logging from optparse arguments
+setup_log_from_args <- function(args) {
```

Comment:
Not sure what you think about this and up to you but is it more useful to have a single log for everything or have an overall log that has one level of messaging (i.e an estimate type has started/ended) and another log that stores the results for that region (like all the run time logs for the USA in a usa.log for example).

Reply:
are you looking for runtimes for each location within the region? If so I would be tempted to output a separate file that details just that in a format that's easy to ingest.
Do you need region level or would an overall run level suffice?
e.g.

report_timings.csv:
| region | subregion | last run timestamp | last duration | n+1 | n+2 | n+3 | n+4 |
| ------- | ----------- | --------------------- | -------------  | ---- | ----- | ----- | ---- |
| USA | Texas | 2020-08-25T11:53:13 | 00:07:23.34 | 00:07:54.23 | 00:05:22.54 | 00:05:09.51 | 00:04:55.90 |
...

Reply:
format of report_timings looks perfect - perhaps with the addition of a mean runtime column. I was more thinking that understanding which reasons are an issue etc is made more difficult by all the internal subnational logging info (which is also useful) so splitting this into an overall log and an internal to each subnational estimate log might be useful - or it might not.

Reply:
perhaps this should all be bumped onto a separate ticket to enhance logging?